.gitignore

```
@@ -5,7 +5,7 @@
 .DS_Store
 
 # Directories that start with _
-_*/
+#_*/   # Unblock to allow for Quarto.  
 !_snaps/
```

```suggestion
_dev/
```

This is really the folder I was targeting anyway, so let's just ignore that one. (I have a convention of always using `_dev/` as a scratchpad, place for notes, reprex files, etc.)

---

I have local TimeMachine backups and keep some of my work in Dropbox... but yeah, anything important in `_dev/` I try to commit or push somewhere else.